create_module.rb

```diff
@@ -0,0 +1,52 @@
+module CreateElement
+  def create_book
+    print 'Title: '
+    title = gets.chomp
+    print 'Author: '
+    author = gets.chomp
+    book = Book.new(title, author)
+    @books.push(book) unless @books.include?(book)
+    puts 'Record is created successfully!!'
+  end
+
+  def create_student
+    print 'Age: '
+    age = gets.chomp
+    print 'Classroom: '
+    classroom = gets.chomp
+    print 'Name: '
+    name = gets.chomp
+    print 'Has Parent Permission (true/false): '
+    has_permission = gets.chomp
+    student = Student.new(name, age.to_i, has_permission, classroom)
+    @person.push(student) unless @person.include?(student)
+    puts 'Record is created successfully!!'
+  end
```

Comment:
- [ ] All your methods should be SOLID 🤔 means they should be responsible for single purpose. But in your code all your methods are having more than one responsibility such as the above method is taking inputs (First Responsibility), creating students (Second Responsibility) 🤔 Kindly make all your methods single responsible 😊